Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Setting default mirror to cdn.debian.net in debian template (Closes: #600464).

diff -Naurp lxc.orig/templates/lxc-debian.in lxc/templates/lxc-debian.in
--- lxc.orig/templates/lxc-debian.in	2010-11-06 21:40:36.574920897 +0100
+++ lxc/templates/lxc-debian.in	2010-11-06 23:21:20.426942404 +0100
@@ -112,7 +112,7 @@ openssh-server
     echo "Downloading debian minimal ..."
     debootstrap --verbose --variant=minbase --arch=$arch \
 	--include $packages \
-	"$SUITE" "$cache/partial-$SUITE-$arch" http://ftp.debian.org/debian
+	"$SUITE" "$cache/partial-$SUITE-$arch" http://cdn.debian.net/debian
     if [ $? -ne 0 ]; then
 	echo "Failed to download the rootfs, aborting."
 	return 1
